src/main/java/ui/listpanel/ListPanelCard.java

```diff
@@ -118,14 +124,23 @@ private Node getEventDisplay(TurboIssue issue, final int withinHours) {
      * @param comments
      * @return
      */
-    private static Node layoutEvents(Model model, TurboIssue issue,
-                                     List<TurboIssueEvent> events, List<Comment> comments) {
+    public static Node layoutEvents(Model model, TurboIssue issue,
```

Comment:
This need not be public just for the sake of testing. There is a way to test private methods using reflection.